tests/system/providers/amazon/aws/example_ec2.py

```diff
@@ -50,11 +50,14 @@ def get_latest_ami_id():
     # on how they name the new images.  This page should have AL2022 info when
     # it comes available: https://aws.amazon.com/linux/amazon-linux-2022/faqs/
     image_prefix = "Amazon Linux*"
+    root_device_name = "/dev/xvda"
 
     images = boto3.client("ec2").describe_images(
         Filters=[
             {"Name": "description", "Values": [image_prefix]},
-            {"Name": "architecture", "Values": ["arm64"]},
+            {"Name": "architecture", "Values": ["x86_64"]},
+            {"Name": "root-device-type", "Values": ["ebs"]},
+            {"Name": "root-device-name", "Values": [root_device_name]},
```

Comment:
Why do we need these changes?

Reply:
Non-graviton instances only support x86 architecture. The root device must be ebs backed in order to support hibernation

Reply:
Hibernation requires an x86 architecture and encrypted storage; encrypted storage requires EBS; EBS requires manually assigning mount points.  It was a bit of a chain of requirements there.

Reply:
Could we add comments? That'd be helpful I think :)

Reply:
Added comments in https://github.com/apache/airflow/pull/35839/commits/4089321bc32e1f5c21246afc3920e8b804aed0a6

Reply:
Thanks a lot @dashton90